src/System.Private.Xml/tests/Xslt/XslCompiledTransformApi/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
+      netfx;
```

Comment:
Did this not require any changes on the .csproj? I would expect at minimum the Configuration names should have changed. Please address it in your next PR.